### ISSUE TITLE
`notify/notify`  Needs documenting, especially how to use the `opts` object 

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#3116]
### Short Summary

The notify module needs some documentation. eg. I can't figure out how to use the `command` option in the `opts` object so I don't know if it's broken or if I'm using it wrong.  

As far as  I can tell I should be able to an opts object like the following:
```
{command:console.log}
```
and when I click on *something* that command will run.  I've tried using it and clicking on the notification when it appears but the command never runs.  I don't know what it's supposed to do without doing a DOM inspection and  searching for the element it appears to set the click handler on:

```
if (notif.opts.command)
        {
            panel.find(".icon, .description").on("click", () => {
                this.hideNotification(notif, false);
                timers.setTimeout(() => {
                    notif.opts.command();
                }, 100);
            });
        }
```

**Comment by Defman21** [comment]
I doubt you can use `console.log` in this way. You either have to wrap it into an anonymous function (`() => console.log("Hello, Carey")`) or `.bind` it. Have you tried that?

**Comment by cgchoffman** [comment]
You don't have to pass args to `console.log` so wrapping it in another function is redundant but I'll def try your suggestion since I've been wrong before :D...
`time passes`
OK so no, you don't need to wrap it in a function BUT I found what you click on!  You click on the text! DUH!.  Thanks for making me do one more test @Defman21.
So the following works and you click on the txt `Komodo is Awesome` to trigger the function:
`require("notify/notify").interact("Komodo is Awesome!","Komodo",{command:console.log})`

Any way, JSDocs should still be written for this stuff.

**Comment by Defman21** [comment]
>Thanks for making me do one more test @Defman21.

You're welcome :)

>You don't have to pass args to console.log

It won't log anything then, will it?

**Comment by cgchoffman** [comment]
> It won't log anything then, will it?

Nope, but that's not the point.  I was trying to get it to work at all.  This isn't a practical implementation, it was testing.

**Comment by Defman21** [comment]
Gotcha. Thanks for explaining that.

**Comment by cgchoffman** [comment]
@Defman21 no problem at all.

**Comment by Naatan** [comment]
> I doubt you can use console.log in this way

Sure you can, it's just a function. Calling it in this way is equivalent to:
```
(arguments) => console.log.apply(console, arguments);
```

**Comment by Defman21** [comment]
Hmm, thanks. I didn't know that.
